### ISSUE TITLE
Accessibility audit for 2024 theme

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/Learn#2482]
- [x] Run scans
- [x] Test with keyboard
- [x] Test with Voiceover
- [x] Test with JAWS/NVDA
- [x] Log issues in comments below
- [x] Fix issues

**Comment by adamwoodnz** [comment]
The search field is within the main landmark on a lot of templates. This should only be the case on the search results page where it is the main functionality. Otherwise it should be above.

**Comment by adamwoodnz** [comment]
See if this is still an issue with the new theme https://github.com/WordPress/wporg-mu-plugins/issues/644

**Comment by adamwoodnz** [comment]
Average a11y scores from automated lighthouse scans are good:

![Image](https://github.com/user-attachments/assets/4d03a51d-aba3-46fd-9262-821cd786b6f5)

Top line is the existing site. Lessons archive doesn't show any issues when manually scanned.

**Comment by adamwoodnz** [comment]
Fallback featured image has no alt text

![Screenshot 2024-07-24 at 1 12 37 PM](https://github.com/user-attachments/assets/fa80512a-44c3-4128-a467-d50686bae6c2)

**Comment by adamwoodnz** [comment]
My Courses card progress bar needs accessible name

![Screenshot 2024-07-24 at 1 16 29 PM](https://github.com/user-attachments/assets/f63390fc-272d-48ed-90aa-198f97ca3ece)

**Comment by adamwoodnz** [comment]
'See all' links don't provide context to screen readers. Should be read as 'See all intermediate learning pathways for WordPress users'

![Screenshot 2024-07-24 at 2 26 20 PM](https://github.com/user-attachments/assets/29769b29-cf18-4b07-ab95-f2d8d02439d3)

**Comment by adamwoodnz** [comment]
> The search field is within the main landmark on a lot of templates. This should only be the case on the search results page where it is the main functionality. Otherwise it should be above.

This is only an issue on the archive pages, where the order of elements is:

1. h1
2. intro paragraph
3. search and filters
4. content (list of courses displayed as cards)

![Screenshot 2024-07-24 at 4 46 33 PM](https://github.com/user-attachments/assets/1c0b07fb-07d8-4d37-a253-8e75a9e2e900)

Currently all of this is within the `main` landmark. On a regular 'single' type document page typically the h1, intro paragraph and content would be in main, with a search field above. However in this case searching and filtering the archive is the core functionality of the page, so it makes sense for them to be located within main with the cards.

**Comment by joedolson** [comment]
In the course cards, all links are duplicated. 

![image](https://github.com/user-attachments/assets/56c6a2af-7dfc-4859-b3e4-c3aa8f8ef791)

While this doesn't cause any immediate problems, it does mean that the travel path is significantly longer for keyboard and screen reader users, and creates a lot of redundancy in navigating course archives. It would be better if these two elements were combined into a single link & the alt text was empty on the image.
